### DISCUSSION TITLE
I am on a mac, and don't understand how to run the darn binaries.

### DISCUSSION THREAD

**Discussion by teacup775** [VSCodium/vscodium#1935] [Q&A] [answered]
I am a little weary of re-installing homebrew (I don't have a problem with brew, just had a tedious mess created from migrating over many computers; cleaning house took days).

vscodium-reh-darwin-arm64-1.90.0.24158 is a set of node executives. this seems a little off

**Comment by daiyam** [comment, marked as answer]
You shoud use [VSCodium.arm64.1.90.2.24171.dmg](https://github.com/VSCodium/vscodium/releases/download/1.90.2.24171/VSCodium.arm64.1.90.2.24171.dmg)

The `reh` binaries are for servers.

**Comment by teacup775** [comment]
Thank ye kindly!
